src/components/ui/ImageGallery/ImageGalleryZoom.tsx

```diff
@@ -0,0 +1,15 @@
+import type { PropsWithChildren } from 'react'
+import React from 'react'
+
+interface ImageGalleryZoomProps {
+  helpMessage?: string
+  zoomFactor?: number
+}
+
+const ImageGalleryZoom = ({
+  children,
+}: PropsWithChildren<ImageGalleryZoomProps>) => {
+  return <> {children} </>
+}
+
+export default ImageGalleryZoom
```

Comment:
Remove this noop component

Reply:
The idea of having this component is just to be a placeholder with some basic structure, so it’s going to be ready to receive its functionality later. With that in mind, we think this should be in a separated file like this because it's not being used right now but it will in the future.

Reply:
Also, the zoom component has value itself. So maybe it could be reused in other structures.

Reply:
Ok, no problem.